lib/jekyll/commands/new.rb

```diff
@@ -75,6 +75,15 @@ def gemfile_contents
 # This is the default theme for new Jekyll sites. You may change this to anything you like.
 gem "minima"
 
+# If you want to develop a custom theme, first create a new theme, like so
+#
+#     bundle exec jekyll new-theme -b minima --name custom-theme _theme
```

Comment:
We call all `jekyll` commands with `bundler` since this is documentation... in a `Gemfile` :smile: